src/Files.App/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -944,6 +945,22 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseItemMenuItems(BaseLayo
                     ShowInZipPage = true,
                 },
                 new ContextMenuFlyoutItemViewModel()
+                {
```

Comment:
It might be better if it was lower down in the menu

Reply:
Below `Tag Section` is ok, or would it be better if I move it in the `More` section?

Reply:
Under the tag section is better than `more`.

Reply:
![Menu](https://user-images.githubusercontent.com/102259289/190449689-eafa1d60-ab0e-4431-b557-b075cccad449.png)

Reply:
👍